README.md

```diff
@@ -1 +1,209 @@
-# React-Group
\ No newline at end of file
+<a name="readme-top"></a>
+
+- [📖 Space Travellers's Hub ](#-space-travellerss-hub-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [INSTALL](#install)
+    - [Usage](#usage)
+    - [Testing](#testing)
+    - [Deployment](#deployment)
```

Comment:
- Hey, since you've not deployed the app, kindly remove the live demo link as its currently broken to make the readme more professional.

Reply:
Removed broken deployed link